packages/components/src/Dialog/DialogActions.tsx

```diff
@@ -12,9 +23,26 @@ export interface DialogActions {
 
 export const DialogActions = ({ variant, size, children }: DialogActions) => {
   const classNames = useClassNames({ component: 'Dialog', variant, size });
+  const { close } = useContext(OverlayTriggerStateContext)!;
+  const closeButtonProps = { onPress: close };
+
   return (
     <div className={cn('[grid-area:actions]', classNames.actions)}>
-      {children}
+      <Provider
+        values={[
+          [
+            ButtonContext,
+            {
+              slots: {
+                [DEFAULT_SLOT]: {},
```

Comment:
what is a default_slot?

Reply:
Needed so you can but a button on there without a slot :) 

https://react-spectrum.adobe.com/react-aria/advanced.html#default-slot